website/docs/automation/gitops/argocd-codecommit/cleanup.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Cleanup"
+sidebar_position: 90
+---
+
+To remove Argo CD configuration for `apps` from the cluster run:
+
+```bash
+$ kubectl -n argocd delete secret codecommit-repo
+secret "codecommit-repo" deleted
+$ kubectl -n argocd delete application apps
```

Comment:
You'll can remove this once you update the reset-environment script as outlined somewhere else in this review.

Reply:
I updated reset-environment, but those resources are created in argocd namespace, so I updated and use explicit clean up here.